ui/src/flux/apis/index.ts

```diff
@@ -43,47 +44,40 @@ export const getAST = async (request: ASTRequest) => {
   }
 }
 
-interface GetTimeSeriesResult {
+export interface GetTimeSeriesResult {
   didTruncate: boolean
   tables: FluxTable[]
+  uuid?: string
 }
 
 export const getTimeSeries = async (
   service: Service,
-  script: string
+  script: string,
+  uuid?: string
 ): Promise<GetTimeSeriesResult> => {
   const mark = encodeURIComponent('?')
-  const garbage = script.replace(/\s/g, '') // server cannot handle whitespace
+  const query = script.replace(/\s/g, '') // server cannot handle whitespace
   const url = `${window.basepath}${
     service.links.proxy
   }?path=/query${mark}organization=defaultorgname`
-  const dialect = {annotations: ['group', 'datatype', 'default']}
-  const data = JSON.stringify({
-    query: garbage,
-    dialect,
-  })
-
   let responseBody: string
   let responseByteLength: number
-
+  let responseUUID: string
   try {
     // We are using the `fetch` API here since the `AJAX` utility lacks support
     // for limiting response size. The `AJAX` utility depends on
     // `axios.request` which _does_ have a `maxContentLength` option, though it
     // seems to be broken at the moment. We might use this option instead of
     // the `fetch` API in the future, if it is ever fixed.  See
     // https://github.com/axios/axios/issues/1491.
```

Comment:
We can probably remove this comment now, since using fetch is no longer novel / exceptional (we use it in a bunch of places).